grails-app/services/org/pih/warehouse/product/ProductPackageService.groovy

```diff
@@ -0,0 +1,62 @@
+package org.pih.warehouse.product
+
+import grails.gorm.transactions.Transactional
+import org.pih.warehouse.core.ProductPrice
+
+@Transactional
+class ProductPackageService {
```

Comment:
I feel somewhat weird about creating product package controller and service, because anyway I have to return `ProductSupplier` from the save method (because I mutate the product supplier also, e.g. `minOrderQuantity`), so I don't know if it's a "REST-y" solution to create a separate controller/service for this case if I do not return the `ProductPackage`, or if it should be handled in the product supplier service.
But since we've discussed to create its own API for that, I decided to go with that, but I can be easily convinced to move it to the `ProductSupplierService` and rename the methods. 
Let me know what you think.